elastic-stack/elasticsearch/configure.sls

```diff
@@ -79,17 +79,32 @@ set_swapiness_for_elasticsearch_node:
 {% endif %}
 
 # Up the count for file descriptors for Lucene https://www.elastic.co/guide/en/elasticsearch/reference/current/file-descriptors.html
-increase_elasticsearch_file_descriptor_limit:
+update_sysctl_file_descriptor_limit:
   cmd.run:
     - name: sysctl -w fs.file-max={{ elasticsearch.fd_limit }}
   file.replace:
     - name: /etc/sysctl.conf
-    - pattern: '^fs.file_max=.*'
-    - repl: fs.file_max={{ elasticsearch.fd_limit }}
+    - pattern: '^fs.file-max=.*'
+    - repl: fs.file-max={{ elasticsearch.fd_limit }}
     - append_if_not_found: True
     - onchanges_in:
         - service: elasticsearch_service
 
+update_elasticsearch_systemd_file_descriptor_limit:
+  file.managed:
+    - name: /etc/systemd/system/elasticsearch.service
```

Comment:
The path for this fragment just needs to be updated and then I think we'll be all set. Should be `/etc/systemd/system/elasticsearch.service.d/fdlimit.conf`

Reply:
Argh, of course. Thanks!

Reply:
Changed in a8a3888